packages/ui/libs/certificate/logic/src/blockchain-inbox/permissions.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  usePermissionsLogic,
+  TUsePermissions,
+  IPermissionReturnType,
+  RequirementList,
+  Requirement,
+} from '@energyweb/origin-ui-certificate-logic';
```

Comment:
You shouldn't use such import paths inside the package itself, it will cause circular dependency error. 
Use relative path `../../someFile`